integration-tools/src/main/java/uk/ac/bbsrc/tgac/miso/integration/dp5mirage/DP5MirageScanner.java

```diff
@@ -65,64 +68,65 @@ public BoxScan getScan() throws IntegrationException{
       // JSON response from Scanner
       response = GetPostParamRequest(httpClient, uri, DP5MirageScanner.params);
 
-      String errorMessage = "Error reported by the scanner.";
-      String actionMessage = "please check this before rescanning.";
-
       // Check if valid before parsing
       if (response.statusCode() != 200) {
         if (response.statusCode() == 453) {
-          throw new IntegrationException(String.format(" %s Container is not found. Check that a "
-              + "container with the"
-              + " container ID: %s is created on the DP5 application", errorMessage,
-              DP5MirageScanner.params.get(
-              "container_uid")));
+          throwScanError(String.format("Container is not found. Check that a container with the "
+                  + "container ID: %s is created on the DP5 application", DP5MirageScanner.params.get(
+                      "container_uid")),
+              true);
         }
         else if (response.statusCode() == 456) {
-          throw new IntegrationException(String.format("%s Scanner type mismatch. %s",
-              errorMessage, actionMessage));
+          throwScanError("Scanner type mismatch", false);
         }
         else if (response.statusCode() == 459) {
-          throw new IntegrationException(String.format("%s The scan is not found. Try rescanning "
-              + "after a short period of time.", errorMessage));
+          throwScanError("The scan is not found. Try rescanning after a short period of time",
+              true);
```

Comment:
This error doesn't make sense. We're telling the scanner to scan and it's saying "scan not found." Do not send specific message to user.